packages/govuk-frontend/src/govuk/components/pagination/pagination.yaml

```
@@ -36,7 +36,11 @@ params:
       - name: text
         type: string
         required: false
-        description: The link text to the previous page. Defaults to 'Previous'.
+        description: The text content of the link to the previous page. If `previous.html` is supplied, this is ignored.
+      - name: html
+        type: string
+        required: false
+        description: The HTML content of the link to the previous page. Defaults to 'Previous page', with 'page' being visually hidden.
```

Similar with this one, think we always add the "this is ignored" caveat to both `text` and `html`?

```suggestion
        description: The HTML content of the link to the previous page. Defaults to 'Previous page', with 'page' being visually hidden. If `previous.text` is supplied, this is ignored.
```

---

I already added this to the batch, but thinking on it, this probably isn't accurate? 

The `text` parameter states that 'If `previous.html` is supplied, this is ignored.' to indicate that the `html` parameter takes priority over the `text` parameter in situations if both are defined. The only situation where `html` is ignored is if it's empty.

Does that need to be worded more clearly? 🤔

---

Yeah weighing everything up, shall we copy what **Summary List** does?

```suggestion
        description: The HTML content of the link to the previous page. Defaults to 'Previous page', with 'page' being visually hidden. If `html` is provided, the `text` option will be ignored.
```

Seems to be the most consistent